src/components/RuleBuilder/Rule.jsx

```diff
@@ -81,21 +78,27 @@ export const Rule = ({
         disabled={disabled}
         options={typeOptions}
         name={`${name}.type`}
-        placeholder={typePlaceholder}
+        placeholder={typeConfig.placeholder}
         label={typeLabel}
         required
+        id={typeConfig.id}
+        showTooltip={typeConfig.tooltip}
```

Comment:
I think you need to pass it as `!!typeConfig.tooltip` ... you use it directly in the component without converting to bool and this will render the tooptip component directly

Reply:
c922df97b7a17a9d3c3c0579897e8a30be5e19c3